holoviews/core/data/grid.py

```diff
@@ -102,18 +102,79 @@ def length(cls, dataset):
 
 
     @classmethod
+    def get_coords(cls, dataset, dim, ordered=False):
+        """
+        Returns the coordinates along a dimension.
+        """
+        data = dataset.data[dim]
+        if ordered and np.all(data[1:] < data[:-1]):
+            data = data[::-1]
+        return data
+
+
+    @classmethod
+    def expanded_coords(cls, dataset, dim):
```

Comment:
Can't this be folded into ``get_coords`` above using an ``expanded=False`` default argument?